content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -42,149 +40,148 @@ To configure AD as the IdP, you must create and modify a Relying Party Trust. Af
 
 6. Modify the newly-created Relying Party Trust.
 
-   a. Select the new SAML Relying Party Trust and then select Properties on the Actions menu displayed on the right side of the screen. The Properties window displays. 
+   a. Select the new SAML Relying Party Trust and then select **Properties** in the **Actions** menu to open the **Properties** window. 
 
    ![SelectRelyingPartyTrustAndProperties](/images/SAML/SelectRelyingPartyTrustAndProperties.png "Select Relying Party Trust And Properties")
         
-   b. Select the Endpoints tab and click **Edit**. The Edit Endpoint window displays.
+   b. Select the **Endpoints** tab, then click **Edit** to open the **Edit Endpoint** window.
 
-   c. Change the Index value to 1 and click **OK**. 
+   c. Change the **Index** value to *1* and click **OK**. 
 
    ![EditPropertiesEndpointsChangeIndexValueCropped](/images/SAML/EditPropertiesEndpointsChangeIndexValueCropped.png "Edit Properties Endpoints Change Index Value Cropped")
        
-   d. Click the **Add SAML** button. The Add Endpoint window displays.
+   d. Click the **Add SAML** button to open the **Add an Endpoint** window.
 
-   e. Type or use copy/paste to input the TrueCommand URL http://IP:PORT/saml/slo in the Trusted URL field.
+   e. Type or copy/paste the TrueCommand URL *http://IP:PORT/saml/slo* in the **Trusted URL** field.
 
    ![EditPropertiesEndpointsEnterTrustedURLSLOCropped](/images/SAML/EditPropertiesEndpointsEnterTrustedURLSLOCropped.png "Edit Properties Endpoints Enter Trusted URL SLO Cropped")
         
    f. Click **OK**, **Apply**, and then **OK**.
 
 7. Configure the Claim Issuance Policy.
 
-   a. Select *Edit Claim Issuance Policy* on the Actions menu. The Edit Claim Issuance Policy for QE SAML window displays. Note the QE SAML is the name you gave your new SAML Relying Party Trust in the preceding steps.
+   a. Select **Edit Claim Issuance Policy** in the **Actions** menu to open the **Edit Claim Issuance Policy for QE SAML** window. The QE SAML is the name you gave your new SAML Relying Party Trust in the preceding steps.
 
    ![EditQESAMLClaim IssuancePolicyWindowCropped](/images/SAML/EditQESAMLClaimIssuancePolicyWindowCropped.png "Edit QE SAML Claim Issuance Policy Window Cropped")
         
-   b. Click **Add Rule** and select *Transform an Incoming Claim*, and then **click Next**. 
+   b. Click **Add Rule** and select **Transform an Incoming Claim**, then click **Next**. 
 
    ![EditClaimIssuancePolicyAddRuleTransformAnIncomingClaimCropped](/images/SAML/EditClaimIssuancePolicyAddRuleTransformAnIncomingClaimCropped.png "Edit Claim Issuance Policy Add Rule Transform An Incoming Claim Cropped")
         
-   c. Click the down arrow for the Incoming claim type field and select Windows account type on the list of displayed options.
+   c. Select **Windows accoiunt name** in the **Incoming claim type** drop-down menu.
```

Comment:
typo

   c. Select **Windows account name** in the **Incoming claim type** drop-down menu.

Reply:
Fixed